lms/djangoapps/teams/static/teams/js/spec/views/teams_tab_spec.js

```diff
@@ -39,7 +57,8 @@ define([
 
         describe('Navigation', function () {
             it('does not render breadcrumbs for the top level tabs', function() {
-                var teamsTabView = createTeamsTabView();
+                var requests = AjaxHelpers.requests(this),
```

Comment:
I don't think you need these requests variables now (here are and all the test cases below).